CliClient/tests/models_Note.js

```diff
@@ -129,4 +135,18 @@ describe('models_Note', function() {
 		expect(duplicatedNote.user_updated_time !== note1.user_updated_time).toBe(true);
 	}));
 
+	it('should delete a set of notebooks', asyncTest(async () => {
+		let folder1 = await Folder.save({ title: 'folder1' });
+		let noOfNotes = 20;
+		for (let i = 0; i < noOfNotes; i++) {
+			await Note.save({ title: `note1${i}`, parent_id: folder1.id });
+		}
+
+		let noteIds = await Folder.noteIds(folder1.id);
+		await Note.batchDelete(noteIds);
+
+		const all = await allItems();
+		expect(all.length).toBe(1);
+	}));
+
```

Comment:
Please add a couple more test cases for `batchDelete`:
1. Delete notes like your test case, but not all of them.  Check that the notes that shouldn't be deleted aren't.
2. Check when noteIds is an empty list ie `batchDelete([])`.  No notes should be deleted.